presto-kafka/src/test/java/io/prestosql/plugin/kafka/encoder/json/TestISO8601JsonDateTimeFormatter.java

```diff
@@ -84,12 +84,12 @@ public void testISO8601DateTimeFunctions()
         testTime(sqlTimeOf(3, 0, 0, 0, 0), 3, "00:00");
         testTime(sqlTimeOf(3, 23, 59, 59, 999000000), 3, "23:59:59.999");
 
-        testTimeWithTZ(sqlTimeWithTimeZoneOf(3, 10, 23, 35, 123000000, 0, 0), "10:23:35.123Z");
-        testTimeWithTZ(sqlTimeWithTimeZoneOf(3, 0, 0, 0, 0, 0, 0), "00:00:00Z");
-        testTimeWithTZ(sqlTimeWithTimeZoneOf(3, 10, 23, 35, 123000000, 2, 0), "10:23:35.123+02:00");
-        testTimeWithTZ(sqlTimeWithTimeZoneOf(3, 10, 23, 35, 123000000, 10, 0), "10:23:35.123+10:00");
-        testTimeWithTZ(sqlTimeWithTimeZoneOf(3, 10, 23, 35, 123000000, -10, 0), "10:23:35.123-10:00");
-        testTimeWithTZ(sqlTimeWithTimeZoneOf(3, 23, 59, 59, 999000000, 0, 0), "23:59:59.999Z");
+        testTimeWithTZ(sqlTimeWithTimeZoneOf(3, 10, 23, 35, 123000000, 0), "10:23:35.123Z");
+        testTimeWithTZ(sqlTimeWithTimeZoneOf(3, 0, 0, 0, 0, 0), "00:00:00Z");
+        testTimeWithTZ(sqlTimeWithTimeZoneOf(3, 10, 23, 35, 123000000, 2 * 60), "10:23:35.123+02:00");
+        testTimeWithTZ(sqlTimeWithTimeZoneOf(3, 10, 23, 35, 123000000, 10 * 60), "10:23:35.123+10:00");
+        testTimeWithTZ(sqlTimeWithTimeZoneOf(3, 10, 23, 35, 123000000, -10 * 60), "10:23:35.123-10:00");
+        testTimeWithTZ(sqlTimeWithTimeZoneOf(3, 23, 59, 59, 999000000, 0), "23:59:59.999Z");
```

Comment:
it's preexisting, but we just learned it's so important

please add a test with minutes offsets here (positive, negative, -00:XX)